### PR TITLE
feat(ui-kit): created library

### PR DESCRIPTION
**Pull request by WilliamChelman**

### CODE REVIEW

File: libs/ui-kit/spinner/spinner.component.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+:host {
+  --size: 20px;
```
Comment (srikanthv02) [inline libs/ui-kit/spinner/spinner.component.scss:2]: 👍

File: libs/ui-kit/src/test-setup.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+import 'jest-preset-angular/setup-jest';
```
Comment (srikanthv02) [inline libs/ui-kit/src/test-setup.ts:1]: can you see this check failure? 🤔

File: libs/ui-kit/spinner/spinner.component.stories.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      description: 'Forefront color of the spinner.',
+      defaultValue: 'var(--cz-color-primary, #007bff)',
+      control: {
+        type: 'color',
+      },
+    },
+    bgColor: {
+      name: 'Background Color ("--color-background" css variable)',
+      description: 'Background color of the spinner.',
+      defaultValue: '#ccc',
```
Comment (srikanthv02) [inline libs/ui-kit/spinner/spinner.component.stories.ts:30]: maybe we can use `var` here too? 🤔

**Review by srikanthv02 — CHANGES_REQUESTED** [review]

File: libs/ui-kit/spinner/spinner.component.stories.ts (hunk 3 above)
Comment (WilliamChelman) [reply libs/ui-kit/spinner/spinner.component.stories.ts:30]: For now, I don't think that a globally defined var would make sense, as it's  acolor specific to the spinner I would say 🤔

**Review by srikanthv02 — APPROVED** [review]

**Pull request merged**